python/tests/test_portfolio.py

```diff
@@ -1,22 +1,22 @@
 # import inspect
-from matplotlib.figure import Figure
-from matplotlib.axes import Axes
-from fastquant import Portfolio
+# from matplotlib.figure import Figure
```

Comment:
@rafmacalaba Just wondering why these tests were commented out?

Reply:
FAILED tests/test_portfolio.py::test_portfolio_init - requests.exceptions.ConnectionError: HTTPConnectionPool(host='1.phisix-api.appspot.com', port=80): Max retries exceeded with url: /stocks/JFC.2020-08-03.json (Caused by...
FAILED tests/test_portfolio.py::test_portfolio_data_query - NameError: name 'p' is not defined
FAILED tests/test_portfolio.py::test_optimization - NameError: name 'p' is not defined

idk why this fails?